File: kata-7kyu/i-m-everywhere.js
function i(word) {
  if(word.length == 0 || word[0].toUpperCase() !== word[0] || word[0] == 'I' 
     || word.replace(/[aeoui]/ig,'').length * 2 <= word.length)
    return "Invalid word"
  return 'i' + word
}

File: python-kata-7kyu/i-m-everywhere.py
import re
def i(word):
    if len(word) == 0 or word[0].upper()!=word[0] or word[0] == 'I' \
    or len(re.sub(r'[AEOUIaeoui]', '', word)) * 2 <= len(word): 
        return 'Invalid word'
    return 'i' + word
